submitted_models/costar_shafter_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,550 @@
+                                <dynamic_bias_correlation_time>300.0</dynamic_bias_correlation_time>
+                            </noise>
+                        </y>
+                        <z>
+                            <noise type="gaussian">
+                                <mean>0</mean>
+                                <stddev>1e-2</stddev>
+                                <bias_mean>0.1</bias_mean>
+                                <bias_stddev>0.001</bias_stddev>
+                                <dynamic_bias_stddev>0.002</dynamic_bias_stddev>
+                                <dynamic_bias_correlation_time>300.0</dynamic_bias_correlation_time>
+                            </noise>
+                        </z>
+                    </linear_acceleration>
+                </imu>
+	    </sensor>
+	    <sensor name="air_pressure" type="air_pressure">
+                <always_on>1</always_on>
+                <update_rate>20</update_rate>
+                <air_pressure>
+                    <reference_altitude>0</reference_altitude>
+                    <noise type="gaussian">
+                        <mean>0.00000008</mean>
+                    </noise>
+                </air_pressure>
+            </sensor>
+            <sensor name="magnetometer" type="magnetometer">
+                <always_on>1</always_on>
+                <update_rate>20</update_rate>
+                <magnetometer>
+                    <x>
+                        <noise type="gaussian">
+                            <mean>0.000000080</mean>
+                            <bias_mean>0.000000400</bias_mean>
+                        </noise>
+                    </x>
+                    <y>
+                        <noise type="gaussian">
+                            <mean>0.000000080</mean>
+                            <bias_mean>0.000000400</bias_mean>
+                        </noise>
+                    </y>
+                    <z>
+                        <noise type="gaussian">
+                            <mean>0.000000080</mean>
+                            <bias_mean>0.000000400</bias_mean>
+                        </noise>
+                    </z>
+                </magnetometer>
+            </sensor>
+
+            <visual name="realsense_front_visual">
+                <pose frame="">0.15 0.0 -0.003 -1.57079632679 0 -1.57079632679</pose>
+                <geometry>
+                    <mesh>
+                        <uri>meshes/zr300.dae</uri>
+                    </mesh>
+                </geometry>
+            </visual>
+          
+            <sensor name="realsense_front" type="rgbd_camera">
+                <pose>0.17 0.0 -0.003 0 0 0</pose>
+                <always_on>1</always_on>
+                <update_rate>30</update_rate>
+                <camera name="realsense_front">
+                    <horizontal_fov>1.18682</horizontal_fov>
+                    <!--lens>
```

Comment:
These parameters must be included for the accuracy in the depth camera optical frame projection.

A set of correct parameters is:
```
<lens>
    <intrinsics>
      <!-- fx = fy = width / ( 2 * tan (hfov / 2 ) ) -->
      <fx>355.815</fx>
      <fy>355.815</fy>
      <!-- cx = ( width + 1 ) / 2 -->
      <cx>240.5</cx>
      <!-- cy = ( height + 1 ) / 2 -->
      <cy>180.5</cy>
      <s>0</s>
  </intrinsics>
</lens>
<distortion>
    <k1>0.0</k1>
    <k2>0.0</k2>
    <k3>0.0</k3>
    <p1>0.0</p1>
    <p2>0.0</p2>
</distortion>
```
![costar_shafter_corrected_rgbd_intrinsics](https://user-images.githubusercontent.com/61025344/104398819-788f4580-551d-11eb-8fab-565c0c170a44.png)

Reply:
the parameters are now updated.